client/src/components/Footer/index.js

```diff
@@ -62,140 +61,94 @@ function Footer() {
               Alumni Network
             </Link>
             <Link to='https://github.com/freeCodeCamp/'>Open Source</Link>
-            <Link to='https://www.freecodecamp.org/news/support/'>Support</Link>
-            <Link to='https://www.freecodecamp.org/news/our-sponsors/'>
-              Sponsors
-            </Link>
-            <Link to='https://www.freecodecamp.org/news/academic-honesty-policy/'>
-              Academic Honesty
-            </Link>
-            <Link to='https://www.freecodecamp.org/news/code-of-conduct/'>
-              Code of Conduct
-            </Link>
-            <Link to='https://www.freecodecamp.org/news/privacy-policy/'>
-              Privacy Policy
-            </Link>
-            <Link to='https://www.freecodecamp.org/news/freecodecamps-terms-of-service/'>
+            <Link to='/news/support/'>Support</Link>
```

Comment:
Should these all news links be external?

Reply:
@raisedadead seems like Gatsby recommend using <a> tag instead of link for external links.
https://www.gatsbyjs.org/docs/gatsby-link/

in that case, should I replace any link that does not route to client with <a> tag?

Reply:
You should audit all the links including the ones in the footer and elsewhere on the static pages that we do serve externally should be external. 

Before that you should also check that we are on the current/correct version of the their implementation.

You should also check that we may have overrides on their implementation.

Finally adding tests would catch these questions.

Reply:
Also I found that some of the links in the footer to `/news` are not current. Can you update them to on this PR?